### ISSUE TITLE
MD5 vs Plaintext passwords

### ISSUE THREAD

**Issue by GiroTheHero** [kinsi55/node_dvripclient#2]
```
//Absolutely stupid custom password hashing. Special thanks to https://github.com/tothi/pwn-hisilicon-dvr#password-hash-function
//We *could* use it, but then again, there isnt really any protection involved with this...
//An attacker can just as well sniff the hash and use that to authenticate, it'd just be an unnecessary extra dependency (md5).
//By checking out the Github link you should come to the conclusion that any device of this kind should *never* be directly
//exposed to the internet anways.
```

Yeah, that's pretty much how that is.

Still, hashing it is mildly a little bit better than plaintext.

Overall, I get the sense that a lot of these "security" devices aren't actually focused on providing security.

Instead, they're more like "false feelings of security" devices.

**Comment by kinsi55** [comment]
Just as mentioned there, I've decided to not implement this feature in favor of not needing the extra md5 dependency. I could possibly make it an optional feature where the user can decide to install the md5 dep on their own and then provide a flag so that its used.

**Comment by kinsi55** [comment]
Implemented in Update 1.1.0
